vue-frontend/package-lock.json

```diff
@@ -2885,7 +2885,6 @@
         "copy-webpack-plugin": "^9.0.1",
         "css-loader": "^6.5.0",
         "css-minimizer-webpack-plugin": "^3.0.2",
-        "cssnano": "^5.0.0",
```

Comment:
We can not directly remove dependency from the package-lock. Need to uninstall package with  `npm uninstall cssnano` command

Reply:
i have used that command only.